package.json

```
@@ -31,7 +31,7 @@
     "lint:test-imports": "node test/find_mismatched_imports.mjs",
     "ci": "npm-run-all lint test check-content-version",
     "check-content-version": "ts-node check-content-version.ts",
-    "coverage": "vitest run --coverage",
+    "coverage": "vitest run --coverage --config vitest.campaign.config.mjs",
```

maybe we can have something like 
```suggestion
    "coverage": "coverage:filtered",
    "coverage:filtered": "vitest run --coverage --config vitest.campaign.config.mjs",
    "coverage:all": "vitest run --coverage",
```
here as well, to not easily forget we do special filtering here